AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -190,4 +198,22 @@ private extension SearchMapView {
             }
         }
     }
+    
+    var finishButton: some View {
+        Button {
+            isSearchAndFollowFlowActive = false
+            tabSelection.selection = .dashboard
+        } label: {
+            Text(Strings.SearchMapView.finishText)
+                .font(Fonts.muliHeading2.bold())
+                .padding(.trailing, 7)
```

Comment:
What would you say to align this well, according to the TextField below; maybe we could use `.padding(. trailing)` just after overlay?